bulk_grades/views.py

```diff
@@ -114,6 +111,25 @@ def post(self, request, course_id, *args, **kwargs):  # pylint: disable=unused-a
                      data.get('waiting', False))
         return HttpResponse(json.dumps(data), content_type='application/json')
 
+    def initialize_processor(self, request, course_id):
```

Comment:
This is missing the `if self.operation_id:` branch from above, meaning exports of error data will no longer work.